GruntFile.js

```diff
@@ -151,10 +151,10 @@ module.exports = function (grunt) {
          },
 
          gitBuild: {
-            command: 'git clone https://github.com/excaliburjs/excalibur-dist build',
+            command: 'git clone -q https://github.com/excaliburjs/excalibur-dist build',
             options: {
                stdout: true,
-               failOnError: true
+               failOnError: false
```

Comment:
You can leave these changes out as master already has this (it doesn't have `-q` but that is just quiet mode).